.github/workflows/ci-pipeline.yml

```diff
@@ -365,16 +365,16 @@ jobs:
     - name: Setup Coq
       run: echo "::add-path::/home/coq/.opam/$COMPILER/bin"
     - name: Build Coq base library
-      run: ./tool/compile-coq.sh base
+      run: ./tool/compile-coq.sh base/coq
```

Comment:
The [corresponding paragraph in the README](https://github.com/FreeProving/free-compiler/blob/736efe5d365aa99d399db5fc7a8557003c468872/README.md#base-library) needs to be updated as well.